kjerneinfo/kjerneinfo-consumer/src/main/java/no/nav/kjerneinfo/consumer/fim/person/vergemal/domain/PdlVerge.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.kjerneinfo.consumer.fim.person.vergemal.domain
+
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.pdl.generated.HentNavnBolk
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.pdl.generated.HentPersonVergemaalEllerFullmakt
+
+open class PdlVerge(
+        private val ident: String?,
+        private val personnavn: HentNavnBolk.Navn?,
+        private val vergesakstype: String?,
+        private val embete: String?,
+        private val omfang: String?,
+        private val gyldighetstidspunkt: HentPersonVergemaalEllerFullmakt.DateTime?,
+        private val opphoerstidspunkt: HentPersonVergemaalEllerFullmakt.DateTime?
+) {
+
+    fun getIdent(): String? {
```

Comment:
Kotlin lager automatisk getter/setter metoder for oss, så vi kan kanskje greie å kvitte oss med disse her.